Update Akka Persistence Cassandra (#1512)
lagom/lagom: docs/manual/scala/guide/cluster/PersistentEntityCassandra.md

With these properties set to `false`, if the keyspaces or tables are missing at startup your service will log an error and fail to start.

> Lagom's Cassandra support is provided by the [`akka-persistence-cassandra`](https://doc.akka.io/docs/akka-persistence-cassandra/0.99/) plugin. A full configuration reference can be in the plugin's [`reference.conf`](https://github.com/akka/akka-persistence-cassandra/blob/v0.99/core/src/main/resources/reference.conf).

Reviewer: ```suggestion
Lagom's Cassandra support is provided by the [`akka-persistence-cassandra`](https://doc.akka.io/docs/akka-persistence-cassandra/0.99/) plugin. A full configuration reference is available in the plugin's [`reference.conf`](https://github.com/akka/akka-persistence-cassandra/blob/v0.99/core/src/main/resources/reference.conf).
```
